pkg/Microsoft.NETCore.Targets/runtime.json

```diff
@@ -0,0 +1,92 @@
+{
+    "supports": {
```

Comment:
Folks who use them would be broken if we removed them.  I felt like removing them was an unnecessary breaking change.